scripts/new_project.kts

```diff
@@ -232,13 +272,13 @@ object NewProject {
         sourceFile.writeText(sourceText)
     }
 
-    private fun showMessage(message: String) {
-        println("\n${message}\n")
-    }
-
-    private fun showErrorMessage(message: String, exitCode: Int = 0) {
+    private fun showMessage(
+        message: String,
+        exitAfterMessage: Boolean = false,
+        exitValue: Int = 0
+    ) {
         println("\n${message}\n")
-        System.exit(exitCode)
+        if (exitAfterMessage) System.exit(0)
```

Comment:
Pretty sure we should pass in the `exitValue` here 😉 
```suggestion
        if (exitAfterMessage) System.exit(exitValue)
```

Reply:
@kaungkhantsoe Last one 💪 💪 💪 💪 🇲🇲